tools/scripts/grpcd/grpcd-ssl-setup.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -euxv
+
+export PS4='+(${BASH_SOURCE}:${LINENO}): ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
+
+SCRIPT_DIR=$(dirname $(readlink -f $0))
+SSL_DIR=${1:-"/opt/directord/grpc-ssl"}
+
+. $SCRIPT_DIR/set-ca.sh
```

Comment:
does this work? $SCRIPT_DIR would be tools/scripts/grpcd, but set-ca.sh and getcert.sh are in tools/scripts/messaging. Should we just move those to be under tools/scripts?

Reply:
ah correct. let me fix

Reply:
I'll move the shared scripts up a level in a follow up commit

Reply:
Actually this was correct because we copy the files into the cwd. So this needs to be where the script is run from.

Reply:
I can symlink the files for consistency if you run it from /usr/share/directord/scripts